script.js

```diff
@@ -0,0 +1,89 @@
+  menu.classList.toggle('fa-times');
+
+  navbar.classList.toggle('active');
+});
+
+window.onscroll = () => {
+  menu.classList.remove('fa-times');
+
+  navbar.classList.remove('active');
+};
+// mobile-end
+
+// testimonial-section-starts
+const speakers = [
+  {
+    name: 'Akshay Mishra',
+    title: 'Associate Software Developer | India',
+    description:
+      ' Adrian, I wanted to thank you for your React courses. I have personally learned a lot from them and landed a full-time job as a JavaScript developer two days ago.',
+    image: './assets/1.png',
+  },
+  {
+    name: 'Akshay Mishra',
+    title: 'Associate Software Developer | India',
+    description:
+      ' Adrian, I wanted to thank you for your React courses. I have personally learned a lot from them and landed a full-time job as a JavaScript developer two days ago.',
+    image: './assets/2.png',
+  },
+  {
+    name: 'Akshay Mishra',
+    title: 'Associate Software Developer | India',
+    description:
+      ' Adrian, I wanted to thank you for your React courses. I have personally learned a lot from them and landed a full-time job as a JavaScript developer two days ago.',
+    image: './assets/3.png',
+  },
+  {
+    name: 'Akshay Mishra',
+    title: 'Associate Software Developer | India',
+    description:
+      ' Adrian, I wanted to thank you for your React courses. I have personally learned a lot from them and landed a full-time job as a JavaScript developer two days ago.',
+    image: './assets/1.png',
+  },
+  {
+    name: 'Akshay Mishra',
+    title: 'Associate Software Developer | India',
+    description:
+      ' Adrian, I wanted to thank you for your React courses. I have personally learned a lot from them and landed a full-time job as a JavaScript developer two days ago.',
+    image: './assets/2.png',
+  },
+  {
+    name: 'Akshay Mishra',
+    title: 'Associate Software Developer | India',
+    description:
+      ' Adrian, I wanted to thank you for your React courses. I have personally learned a lot from them and landed a full-time job as a JavaScript developer two days ago.',
+    image: './assets/3.png',
+  },
+];
+
+for (let i = 0; i < speakers.length; i += 1) {
+  const data = document.createElement('div');
+  data.classList.add('testimonials');
+  data.innerHTML = `
+                <div class="students-testimonials">
+                <div class="testimonial-bg">
+                <img src="${speakers[i].image}" alt=" " />
+                </div>
+
+                <div class="student-info">
+                    <h3 class="student-name">
+                    ${speakers[i].name}
+                    </h3>
+                    <span class="student-position">
+                    ${speakers[i].title}
+                    </span>
+                    <hr />
+                    <p class="testimonial">
+                    ${speakers[i].description}
+                    </p>
+                </div>
+                </div>
```

Comment:
- [ ] As requested by the previous reviewer, it would be nicer to adjust the margins of the speakers' section to match that of the design template as much as possible. I recommend aligning this section to the center of the page to improve your UI design.
- [ ] The header's left and right margins should match what was specified in the original design. It would be nicer if the left and right margins have the same value.

- Your design:
<img width="703" alt="image" src="https://user-images.githubusercontent.com/77190203/175785040-b733a427-b81c-4ec0-8812-647c48134c46.png">

- Original design:
<img width="615" alt="image" src="https://user-images.githubusercontent.com/77190203/175785110-d13051f1-f680-48e7-bd1f-3b8ae74fe159.png">

<img width="616" alt="image" src="https://user-images.githubusercontent.com/77190203/175785066-fcfed2fc-3e84-4e8d-9783-83e126940ae4.png">